hf_repo_exporter.py

```diff
@@ -0,0 +1,475 @@
+            last_modified = last_modified.replace(tzinfo=timezone.utc)
+
+        one_year_ago = datetime.now(timezone.utc) - timedelta(days=365)
+        return "Yes" if last_modified < one_year_ago else "No"
+    except Exception:
+        return "N/A"
+
+def get_model_card_field(repo, key: str) -> str:
+    try:
+        value = repo.card_data.get(key, "")
+        # Convert to string if it's a list or other type
+        if isinstance(value, list):
+            return ", ".join(str(v) for v in value)
+        return str(value) if value else ""
+    except Exception:
+        return "N/A"
+    
+def get_associated_datasets(repo) -> str:
+    try:
+        datasets = [tag.replace("dataset:", "") for tag in repo.tags if tag.startswith("dataset:")]
+        return ", ".join(datasets) if datasets else "No"
+    except Exception:
+        return "No"
+    
+def get_associated_models(api, repo, repo_type) -> str:
+    found = [tag.replace("model:", "") for tag in getattr(repo, "tags", []) if tag.startswith("model:")]
+    
+    # If it's a dataset, search for models that use this dataset
+    if repo_type == "dataset":
+        try:
+            related_models = api.list_models(filter=f"datasets:{repo.id}")
+            for m in related_models:
+                if m.id not in found:
+                    found.append(m.id)
+        except Exception:
+            pass
+            
+    return ", ".join(found) if found else "No"
+    
+def get_associated_spaces(api, repo, repo_type) -> str:
+    # 1. Check direct tags in the model card (Upstream)
+    found = [tag.replace("space:", "") for tag in getattr(repo, "tags", []) if tag.startswith("space:")]
+    
+    # 2. If it's a model, search for Spaces that use this model (Downstream)
+    if repo_type == "model":
+        try:
+            # Search for spaces that list this model ID
+            related_spaces = api.list_spaces(filter=f"models:{repo.id}")
+            for s in related_spaces:
+                if s.id not in found:
+                    found.append(s.id)
+        except Exception:
+            pass
+            
+    return ", ".join(found) if found else "No"
+
+def get_doi(repo) -> str:
+    try:
+        for tag in repo.tags:
+            if tag.startswith("doi:"):
+                return tag.replace("doi:", "")
+    except Exception:
+        pass
+
+    return "No"
+
+def extract_link_from_text(text, label):
+    if not text:
+        return "No"
+
+    # Pattern to find 'Label: ...' anywhere in the text
+    # Added \b to ensure we match the exact word
+    pattern = rf"\b{label}\b:\s*([^\r\n]+)"
+    match = re.search(pattern, text, re.IGNORECASE)
+    
+    if match:
+        content = match.group(1).strip()
+        # Remove common markdown junk: *, _, `, [, ]
+        content = re.sub(r'[*_`\[\]]', '', content).strip()
+        
+        # Filter out placeholders
+        if content.upper() in ["N/A", "NONE", "", "NULL", "TBA", "COMING SOON", "IN PROGRESS", "-->"]:
```

Comment:
```suggestion
        if content.upper() in ["N/A", "NONE", "", "NULL", "TBA", "COMING SOON", "IN PROGRESS", "TBD","-->"]:
```
`"-->"` renders when there's a portion commented out, so maybe have if it contains?